### PR TITLE
cubic scaling Laplace-SOS-MP2

### PR DESCRIPTION
**Pull request by fstein93**
I used the code of the cubic scaling RPA to enable cubic scaling Laplace-SOS-MP2. Therefore, I changed the input a bit and added regtest.

### CODE REVIEW

**Comment by dev-zero** [comment]
please do a rebase

**Comment by fstein93** [comment]
That is strange. The only failing regtest (QS/regtest-gpw-5/si7c_broy_gapw_a04_restart.inp) works fine on my machine (I get -32.70494373736452.). Interestingly, it shouldn't be affected at all because I changed files in the RPA part.

**Comment by alazzaro** [comment]
Is your branch in sync with the master? I see that Ole made some "tests stabilizations". 
Anyway, I think this is false positive, so I would proceed to merging...

File: src/rpa_im_time.F [hunk 1]
```diff
@@ -150,6 +150,7 @@ MODULE rpa_im_time
 !> \param cell_to_index_3c ...
 !> \param needed_cutRI_mem_R1vec_R2vec_for_kp ...
 !> \param has_mat_P_blocks ...
+!> \param do_ri_sos_laplace_mp2 wether we perform ri-laplace-sos mp2
```
Comment (dev-zero) [inline src/rpa_im_time.F]: `wether` -> `whether`

File: src/rpa_ri_gpw.F [hunk 2]
```diff
@@ -2193,8 +2192,9 @@ SUBROUTINE rpa_num_int(qs_env, Erpa, mp2_env, para_env, para_env_RPA, para_env_s
       TYPE(two_dim_int_array), ALLOCATABLE, &
          DIMENSION(:, :)                                 :: offset_combi_block
 
-      CALL timeset(routineN, handle)
+!, mat_P_omega_im_part_beta
```
Comment (dev-zero) [inline src/rpa_ri_gpw.F]: ???

**Pull request merged**